fleece/cli/run/run.py

```diff
@@ -179,19 +196,13 @@ def run(args):
     env['AWS_SECRET_ACCESS_KEY'] = aws_credentials['secretAccessKey']
     env['AWS_SESSION_TOKEN'] = aws_credentials['sessionToken']
 
-    process = subprocess.Popen(
-        args.command,
-        env=env,
-        shell=True,
-        stderr=subprocess.STDOUT,
-        stdout=subprocess.PIPE,
-        universal_newlines=True,
-    )
-    for line in iter(process.stdout.readline, ''):
-        sys.stdout.write(line)
+    if isinstance(args.command, list):
+        command = ' '.join("'{}'".format(c.replace("'", "\\'"))
+                           for c in args.command)
+    else:
+        command = args.command
 
-    return_code = process.wait()
-    sys.exit(return_code)
+    sys.exit(subprocess.call(command, shell=True, env=env))
```

Comment:
The solution based in `Popen` that you replaced had the benefit that the output of the command was sent to `stdout`, which means that the build log would have it. With this change the output of the command is lost, I believe.

Reply:
`subprocess.call` also sends process's `stdout`.

Reply:
Okay, then I have no problem with this. I seem to recall we had problems with this, maybe I'm confusing it with something else. We also had problems trying to get the output of the `docker build` command to appear on our builds, maybe that was what needed weird workarounds.